openstudio-extension.gemspec

```diff
@@ -41,12 +35,12 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'octokit', '~> 4.18.0' # for change logs
   spec.add_dependency 'openstudio_measure_tester', '~> 0.5.0'
   spec.add_dependency 'openstudio-workflow', '~> 2.5.0'
-  # parallel, regexp_parser, and addressable versions are pinned to avoid test_with_openstudio errors
-  spec.add_dependency 'addressable', '>= 2.8.1'
+  # parallel, regexp_parser, parser, and addressable versions are pinned to avoid test_with_openstudio errors
+  spec.add_dependency 'addressable', '2.8.1'
   spec.add_dependency 'parallel', '~> 1.19.1'
-  spec.add_dependency 'regexp_parser', '>= 2.9.0'
+  spec.add_dependency 'parser', '< 3.3.0'
```

Comment:
That should be pinned exactly, I think you want

  spec.add_dependency 'parser', '3.2.2.2'


https://github.com/NREL/OpenStudio-measure-tester-gem/pull/67

Also, I find it weird that you have to add it, since the openstudio_measure_tester has it. I think the issue is that the openstudio measure test was broken recently when it updated the gems there.

@anchapin

Reply:
* https://github.com/NREL/OpenStudio-measure-tester-gem/issues/66

Reply:
@jmarrec what's the best solution for this? Should I make a new version 0.9.7 for this change or just make note of this and fix it when making 0.9.7? Also, it sounds like you indicated that the 'parser' dependency isn't need to be listed here once it is fixed in measure-tester-gem.

@kflemin @DavidGoldwasser